src/keyboard.cpp

```diff
@@ -0,0 +1,196 @@
+#include "keyboard.h"
+#include "aldente.h"
+#include "physics.h"
+#include "scene/scene_camera.h"
+
+bool Keyboard::keys[1024];
+bool Keyboard::lmb_down;
+bool Keyboard::rmb_down;
+bool Keyboard::mouse_moved;
+glm::vec3 Keyboard::last_cursor_pos;
+
+// TODO : This class won't be used in the final game anyways.
+const GLfloat BASE_CAM_SPEED = 0.1f;
```

Comment:
Please qualify with static if these are not meant to be globally visible.

Also, we should separate input logic from camera logic. Consider a separate `Camera` module, and we can lump in keyboard and mouse stuff into our existing gamepad-handling `input` module.